webpack.prod.ts

```diff
@@ -31,17 +31,13 @@ module.exports = mergeWebpack(commonWebpack, {
   },
   optimization: {
     noEmitOnErrors: true,
+    minimize: true,
     minimizer: [
       new TerserJSPlugin({
-        cache: true,
         parallel: 2,
-        sourceMap: true,
       }),
       new OptimizeCSSAssetsPlugin({}),
     ],
-    splitChunks: {
```

Comment:
Double checking. The webpack merge is a deep merge, correct? Because the configuration of `chunks: 'all'` is a [good thing](https://webpack.js.org/plugins/split-chunks-plugin/#splitchunkschunks) we want to make sure stays in.

Reply:
@wiedld thanks for the review! I'm almost certain that the prod config will inherit `splitChunks: 'all'` from the common config, but I will double check it.

I'll also look at the bundle size analysis. Good suggestions, love where your head is.